tests/multi_asic_intfutil_test.py

```diff
@@ -0,0 +1,146 @@
+-----------  ------  -------  -----------  ------------------------
+  Ethernet0      up       up  Ethernet1/1  ARISTA01T2:Ethernet3/1/1
+  Ethernet4      up       up  Ethernet1/2  ARISTA01T2:Ethernet3/2/1
+"""
+
+intf_description_asic0_all = """\
+   Interface    Oper    Admin         Alias               Description
+------------  ------  -------  ------------  ------------------------
+   Ethernet0      up       up   Ethernet1/1  ARISTA01T2:Ethernet3/1/1
+   Ethernet4      up       up   Ethernet1/2  ARISTA01T2:Ethernet3/2/1
+Ethernet-BP0      up       up  Ethernet-BP0          ASIC1:Eth0-ASIC1
+Ethernet-BP4      up       up  Ethernet-BP4          ASIC1:Eth1-ASIC1
+"""
+class TestInterfacesMultiAsic(object):
+    @classmethod
+    def setup_class(cls):
+        print("SETUP")
+        os.environ["PATH"] += os.pathsep + scripts_path
+        os.environ["UTILITIES_UNIT_TESTING"] = "3"
+
+    def setUp(self):
+        self.runner = CliRunner()
+
+
+    def test_multi_asic_interface_status_all(self):
+        result = subprocess.check_output(
+            'intfutil -c status -d all', stderr=subprocess.STDOUT, shell=True)
+        print(result)
+        assert result == intf_status_all
+
+    def test_multi_asic_interface_status(self):
+        result = subprocess.check_output(
+            'intfutil -c status', stderr=subprocess.STDOUT, shell=True)
+        print(result)
+        assert result == intf_status
+
+    def test_multi_asic_interface_status_asic0_all(self):
+        result = subprocess.check_output(
+            'intfutil -c status -n asic0 -d all', stderr=subprocess.STDOUT, shell=True)
+        print(result)
+        assert result == intf_status_asic0_all
+
+    def test_multi_asic_interface_status_asic0(self):
+        result = subprocess.check_output(
+            'intfutil -c status', stderr=subprocess.STDOUT, shell=True)
+        print(result)
+        assert result == intf_status_asic0
+
+    def test_multi_asic_interface_desc(self):
+        result = subprocess.check_output(
+            'intfutil -c description', stderr=subprocess.STDOUT, shell=True)
+        print(result)
+        assert result == intf_description
+
+    def test_multi_asic_interface_desc_all(self):
+        result = subprocess.check_output(
+            'intfutil -c description -d all', stderr=subprocess.STDOUT, shell=True)
+        print(result)
+        assert result == intf_description_all
+
+    def test_multi_asic_interface_asic0(self):
+        result = subprocess.check_output(
+            'intfutil -c description -n asic0', stderr=subprocess.STDOUT, shell=True)
+        print(result)
+        assert result == intf_description_asic0
+
+    def test_multi_asic_interface_desc_asic0_all(self):
+        result = subprocess.check_output(
+            'intfutil -c description -n asic0 -d all', stderr=subprocess.STDOUT, shell=True)
+        print(result)
+        assert result == intf_description_asic0_all
+        
+        
+    def teardown_class(cls):
```

Comment:
i do not see any negative unit test cases, for example, check an asic name that does not exists.

Reply:
add negative cases for invalid asic-id